### PR TITLE
XW-2625 | Fix tests for CMP

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/XW-2625

After we migrated to not routable sections some tests are not valid anymore. Others required updates.

Ping @qaga

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/NavigationTests.java [hunk 1]
```diff
@@ -40,26 +39,37 @@ private void init() {
...
 
     UrlChecker.isPathContainedInCurrentUrl(
-        driver, MercuryPaths.ROOT_PATH_SECTION + curatedContent.getTitle());
+        driver, MercurySubpages.CC_MAIN_PAGE);
   }
 
-  @Test(groups = "MercuryCuratedNavigationTest_002")
-  public void MercuryCuratedNavigationTest_002_redirectToExistingArticle() {
+  @Test(groups = "MercuryCuratedNavigationTest_003")
+  public void MercuryCuratedNavigationTest_003_redirectToExistingArticle() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/NavigationTests.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/NavigationTests.java (hunk 1 above)
Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/NavigationTests.java]: LGTM

**Comment by qaga** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**